src/componentTypesView.ts

```diff
@@ -4,52 +4,38 @@
  *-----------------------------------------------------------------------------------------------*/
```

Comment:
There should be no dramatic changes in this file. There should be command to open webview based Registry View and that is it.